README.md

```diff
@@ -49,6 +49,15 @@ computational effects.
 
 ### 2017
 
+* **Structured asynchrony with algebraic effects** (TyDe 2017)  
+  by Daan Leijen  
+  ([pdf](https://www.microsoft.com/en-us/research/wp-content/uploads/2017/05/asynceffects-msr-tr-2017-21.pdf))
+
+* **Implementing Algebraic Effects in C** (APLAS 2017)  
+  by Daan Leijen  
+  ([pdf](https://www.microsoft.com/en-us/research/wp-content/uploads/2017/06/algeff-in-c-tr-v2.pdf))
```

Comment:
This one's another TR, not an APLAS paper.

Reply:
Right. It is however accepted https://www-aplas.github.io/program.html

Reply:
Glad to hear it!  If you change the label to `([Tech Report](…))` I'm happy to merge.